tutorial/file-formats.md

```diff
@@ -16,7 +16,7 @@ commonly used ones.
 ### MIME
 Example: `text/lua`, `image/sketch`, `silica/theme`
 
-Every file format must be given ***one*** MIME. If more than one already exists pick **one**.
+Every file format must be given excatly one MIME. If more than one already exists pick **one**.
```

Comment:
I hate to nitpick again, but I don't think that's *exactly* how you spell that word. ;-)